src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,300&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Product cards</h1>
+
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__items"
+        src="./images/imac.jpeg"
+        alt="Imac photo"
+        width="160px"
+        height="134px"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+      <div class="grade">
+        <div class="grade__stars">
+          <div class="grade__stars-star"></div>
+          <div class="grade__stars-star"></div>
+          <div class="grade__stars-star"></div>
+          <div class="grade__stars-star"></div>
+          <div class="grade__stars-star"></div>
+        </div>
+        <p>Reviews: 5</p>
```

Comment:
The text 'Reviews: 5' should be wrapped in a span or similar element to separate it from the `p` tag used for the number of reviews, which allows for more flexible styling and better accessibility.